packages/atlas/src/components/_auth/AuthModals/AuthModals.tsx

```diff
@@ -1,18 +1,22 @@
 import { ExternalSignInModal } from '@/components/_auth/ExternalSignInModal'
 import { LogInModal } from '@/components/_auth/LogInModal'
 import { SignUpModal } from '@/components/_auth/SignUpModal'
+import { useDebounceValue } from '@/hooks/useDebounceValue'
 import { useAuthStore } from '@/providers/auth/auth.store'
 
 export const AuthModals = () => {
   const { authModalOpenName } = useAuthStore()
-  switch (authModalOpenName) {
-    case 'logIn':
-      return <LogInModal />
-    case 'externalLogIn':
-      return <ExternalSignInModal />
-    case 'signUp':
-      return <SignUpModal />
-    default:
-      return null
+  const debouncedAuthModalOpenName = useDebounceValue(authModalOpenName, 2_000)
```

Comment:
I would remove that, it doesn't work well, IMO it's worse than before

Reply:
New approach in commit `CR fixes`